README.md

```diff
@@ -469,3 +469,23 @@ provider := vcsutils.GitHub
 
 webhookInfo, err := webhookparser.ParseIncomingWebhook(provider, token, request)
 ```
+
+### Upload Code Scanning
```

Comment:
```suggestion
#### Upload Code Scanning
```

Additionally, please move this API above the Webhook parser

Reply:
Added